src/client/menu/users/components/user-form/index.js

```diff
@@ -42,7 +42,7 @@ export default class UserForm extends React.Component {
       id: user && user.id,
       name: this.state.name || user.name,
       email: this.state.email || user.email,
-      rsaKeyPath: this.state.rsaKeyPath || user.rsaKeyPath,
+      rsaKeyPath: this.state.rsaKeyPath || (user && user.rsaKeyPath),
```

Comment:
Should we add this `user &&` to the two above as well? Just a thought.

Reply:
So this form is used on the create and update. I think we don't want to support null names/emails, so it might be good that it blows up? We could probably still use some field verification on those though.